src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/dto/offendersearch/OffenderSearchPrisoner.kt

```diff
@@ -5,7 +5,7 @@ data class OffenderSearchPrisoner(
   val firstName: String,
   val lastName: String,
 
-  val prisonId: String,
-  val prisonName: String,
+  val prisonId: String?,
+  val prisonName: String?,
   val cellLocation: String?,
```

Comment:
this is the main change: all location-type fields are nullable. the rest of the changes in this commit propagate these nulls

Reply:
Wouldn't this have an impact on (other) clients tho? It sounds like a breaking change (although a rare one as I'd imagine most of the data _will_ have the prisoner number)